lib/reporting/identity_verification_report.rb

```diff
@@ -149,7 +151,8 @@ def usps_enrollment_status_updated
     end
 
     def successfully_verified_users
-      idv_final_resolution_verified + gpo_verification_submitted + usps_enrollment_status_updated
+      idv_final_resolution_verified + gpo_verification_submitted + usps_enrollment_status_updated +
+        fraud_review_passed
```

Comment:
In CloudWatch, I see that the `Fraud: Profile review passed` rows have the computed value `identity_verified` set to 1, so I checked  whether adding the `fraud_review_passed` count here will double count them. Looks like `identity_verified` is only used for the final_resolution event, so the code should work as is. Tests pass.